cmd/create.go

```diff
@@ -106,7 +121,7 @@ func runCreate(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("failed to push image: %s", err)
 		}
 	}
-	stackMeta.Update(createTag, stackService)
+	stackMeta.Update(createTag, stackTags, sliceName, stackService)
```

Comment:
this probably should have an error check

Reply:
Added